File: 0771-jewels-and-stones/0771-jewels-and-stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int res = 0;
        vector<int> js(128);
        for(char j : jewels)
            ++js[j];
        for(char s : stones)
            res += (js[s] > 0);
        return res;
    }
};

File: 0771-jewels-and-stones/0771-jewels-and-stones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        int count = 0;
        int[] js = new int[128];
        for(char j : jewels.toCharArray()) {
            js[j - '0']++;
        }
        for(char s : stones.toCharArray()) {
            if(js[s- '0'] > 0) {
                ++count;
            }
        }
        return count;
    }
}